### PR TITLE
Adjust form control line height to fit descenders

### PR DESCRIPTION
**Pull request by johnhwhite**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-input-line-height](https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-input-line-height)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-input-line-height](https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-input-line-height)

**Review by Blackbaud-AlexKingman — DISMISSED** [review]
Looks good, pending UX approval.

**Comment by Blackbaud-ToddRoberts** [comment]
@Blackbaud-AdamFunderburk do we need to do anything to compensate for this making input boxes 57px tall rather than 55px?

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-input-line-height](https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-input-line-height)

**Comment by Blackbaud-ToddRoberts** [comment]
@Blackbaud-TrevorBurch will the change in input box height this causes lead to any issues/inconsistencies with superselect/lookup?

**Comment by Blackbaud-AdamFunderburk** [comment]
> @Blackbaud-AdamFunderburk do we need to do anything to compensate for this making input boxes 57px tall rather than 55px?

I checked with @Blackbaud-TrevorBurch about the position of Tokens inside an Input Box and it looks like we'll need to move them down 1px to stay aligned with the new value text.

We saw some differences in the position of the new value text and placeholder text. Is this new line height supposed to be applied to placeholder text as well?

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-input-line-height](https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-input-line-height)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-input-line-height](https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-input-line-height)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-input-line-height](https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-input-line-height)

**Comment by johnhwhite** [comment]
> > @Blackbaud-AdamFunderburk do we need to do anything to compensate for this making input boxes 57px tall rather than 55px?
> 
> I checked with @Blackbaud-TrevorBurch about the position of Tokens inside an Input Box and it looks like we'll need to move them down 1px to stay aligned with the new value text.
> 
> We saw some differences in the position of the new value text and placeholder text. Is this new line height supposed to be applied to placeholder text as well?

@Blackbaud-ToddRoberts @Blackbaud-AdamFunderburk I've adjusted the padding to compensate for the height and that reduced the number of visual regression issues. HTML placeholder should pick up the style adjustment. 

I'm checking with @Blackbaud-TrevorBurch on the tokens issue.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-input-line-height](https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-input-line-height)

File: src/app/public/modules/input-box/input-box.component.scss [hunk 1]
```diff
@@ -1,6 +1,7 @@
 @import "~@skyux/theme/scss/mixins";
 @import "~@skyux/theme/scss/_compat/mixins";
 @import "~@skyux/theme/scss/variables";
+@import "~@skyux/theme/scss/_compat/variables";
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/input-box/input-box.component.scss:4]: Is this import necessary?

**Review by Blackbaud-AdamFunderburk — APPROVED** [review]

**Pull request merged**